lib/flare_animation.dart

```diff
@@ -0,0 +1,133 @@
+import 'dart:ui';
+import 'dart:math';
+
+import "flame.dart";
+import "package:flare_flutter/flare.dart";
+import "package:flare_flutter/flare_actor.dart";
+
+class FlareAnimation {
+  final FlutterActorArtboard _artboard;
+
+  String _animationName;
+  final List<FlareAnimationLayer> _animationLayers = [];
+
+  double x = 0.0,
+      y = 0.0,
+      _width = 0.0,
+      _height = 0.0,
+      xScale = 0.0,
+      yScale = 0.0;
+
+  Picture _picture;
+
+  FlareAnimation(this._artboard);
+
+  static load(String fileName) async {
+    final actor = FlutterActor();
+    await actor.loadFromBundle(Flame.bundle, fileName);
+
+    final artboard = actor.artboard.makeInstance();
+    artboard.makeInstance();
+    artboard.initializeGraphics();
+    artboard.advance(0.0);
+
+    return FlareAnimation(artboard);
+  }
+
+  double get width {
+    return _width;
+  }
+
+  double get height {
+    return _height;
+  }
+
+  set width(double newWidth) {
+    _width = newWidth;
+    xScale = _width / _artboard.width;
+  }
+
+  set height(double newHeight) {
+    _height = newHeight;
+    yScale = _height / _artboard.height;
+  }
+
+  void updateAnimation(String animation) {
+    _animationName = animation;
+
+    if (_animationName != null && _artboard != null) {
+      _animationLayers.clear();
+
+      ActorAnimation animation = _artboard.getAnimation(_animationName);
+      if (animation != null) {
+        _animationLayers.add(FlareAnimationLayer()
+          ..name = _animationName
+          ..animation = animation
+          ..mix = 1.0
+          ..mixSeconds = 0.2);
+        animation.apply(0.0, _artboard, 1.0);
+        _artboard.advance(0.0);
+      }
+    }
+  }
+
+  @override
```

Comment:
yet it overrides render and update? from whom?
I might be missing something obvious here...

Reply:
This override was wrong, I removed it.